releasenotes/notes/1.1/storage-var-a00a33fcf9a71f3f.yaml

```diff
@@ -6,7 +6,7 @@ features_circuits:
 
     See :ref:`circuit-real-time-methods` for more in-depth discussion on all of these.
 
-    The new methods are:
+    The new methods are::
```

Comment:
I don't think this is correct. The double `::` is used to start a literal block: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks instead you wanted a bulleted list (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks). That being said this note file needs to be delete it's from 1.1.